### ISSUE TITLE
Improve URL comparison at urlSelector by ignoring prefixes

### ISSUE THREAD

**Issue by IAmKontrast** [timo95/stash-checker#40]
URL string comparison could fail even it's basically the same address cased by different prefixes like 'http', 'https', and 'www.'.

http://example.com/video/1234
https://example.com/video/1234
http://www.example.com/video/1234
https://www.example.com/video/1234

All four head to the same scene but the URL selector cannot match correctly with our Stash data when the website changes over the time.

We could ignore the prefixes, or do you have a better idea?

**Comment by timo95** [comment]
I guess it would be possible to hardcode some variants or a simple regex in the GraphQL query. But everything is moving to https and the www subdomain has been going out of fashion for a long time, so this should be rare.

Other variants like [not] including the title in the URL are more common. Most of the time only one or two possible ways are actually used though, so multiple static URLs (atm via multiple check() calls) are enough for me. I do want to refactor check in the future to accept multiple selectors of the same type, which would make this less janky.

**Comment by IAmKontrast** [comment]
Rare but happens. Several scenes have either 'www.' or bare url (or both) at Stashdb and ThePornDB. I don't wanna get a cross mark on the media pages although I got the scene because of a missing (or existing) 'www.' in the url in my Stash when I've added it.

Multiple URLs would be a charm especially for the page hobby.porn which contains the _own_ url and pornhub url too.

I found another - for me - acceptable solution for the (naked) url issue:
Get the `naked domain` and compare with `includes modifier` instead of `equals` ... #41
